Projects/Contracts/Sources/Journeys/ContractInformation.swift

```diff
@@ -27,46 +27,35 @@ struct ContractInformationView: View {
                         }
                         .sectionContainerStyle(.transparent)
                     }
-                    if let table = contract.currentAgreementsTable {
-                        ForEach(table.sections) { section in
-                            hSection(section.rows, id: \.title) { row in
-                                hRow {
-                                    hText(row.title)
+                    VStack {
+                        if let table = contract.currentAgreementsTable {
+                            ForEach(table.sections) { section in
+                                hSection(section.rows, id: \.title) { row in
```

Comment:
@sladan-hedvig for some reason the view is a bit off here, the divider is not aligning